jax/_src/scipy/stats/_core.py

```diff
@@ -101,7 +110,20 @@ def rankdata(
   axis: int | None = None,
   nan_policy: str = "propagate",
 ) -> Array:
+  """Compute the rank of data along an array axis.
```

Comment:
I read this and still wasn't sure what the function does. I had to read scipy's docs to figure it out.

I suspect this needs a couple more sentences at least.

Ideally we'd include an example for every function, as well.

I admit that we probably don't want to completely replicate scipy's docs, but I think we probably need to do slightly more so these docs can be read in isolation.